qdyn/pyqdyn.py

```diff
@@ -937,28 +939,39 @@ def read_output(self, mirror=False, path_output=None, read_ot=True, filename_ot=
 
             # List where each element is a DataFrame with the output of one fault
             self.fault = [None] * N_faults
+            print(self.fault)
```

Comment:
This is probably just for debugging? If yes: remove

Reply:
yes, removed